ui/component/or-attribute-card/src/index.ts

```diff
@@ -131,21 +221,24 @@ export class OrAttributeCard extends LitElement {
     @query("#chart")
     private _chartElem!: HTMLCanvasElement;
     private _chart?: Chart;
-    private _style!: CSSStyleDeclaration;
 
     static get styles() {
         return [
+            css`${unsafeCSS(dialogStyle)}`,
             style
         ];
     }
 
     connectedCallback() {
         super.connectedCallback();
-        this._style = window.getComputedStyle(this);
+
+        if (!this.assetId || !this.attributeName) { return false; }
+
```

Comment:
These properties can be changed during the lifecycle of the component so shouldn't be checked here but in `shouldUpdate` or `updated`

Reply:
`connectedCallback` and `disconnectedCallback` should be used for attaching/detaching event handlers etc. to prevent memory leaks